docker/.env

```
@@ -264,6 +264,8 @@ SMQ_PASSWORD_RESET_URL_PREFIX=http://localhost/password-reset
 SMQ_PASSWORD_RESET_EMAIL_TEMPLATE=reset-password-email.tmpl
 SMQ_VERIFICATION_URL_PREFIX=http://localhost/verify-email
 SMQ_VERIFICATION_EMAIL_TEMPLATE=verification-email.tmpl
+SMQ_USERS_INVITATION_SENT_EMAIL_TEMPLATE=invitation-sent-email.tmpl
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMQ_USERS_INVITATION_SENT_EMAIL_TEMPLATE key should go before the SMQ_USERS_LOG_LEVEL key

(UnorderedKey)